h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -70,12 +71,16 @@ export default function AnnotationCard({
         await updateModerationStatus(status);
         onStatusChange(status);
         setModerationSaveState({ type: 'saved' });
-      } catch {
-        // TODO Catching all errors generally for now, but we need to explicitly
-        //      handle conflict errors
+      } catch (e) {
+        const isConflictError =
+          e instanceof APIError && e.response?.status === 409;
+        const error = isConflictError
+          ? 'The annotation has been updated by its author'
```

Comment:
This message is more or less a placeholder. We can refine it based on feedback and once other related issues have been addressed.

For example, once we make the annotation data be updated automatically after the error has occurred, we'll probably want to indicate the info is now up to date and it's safe to retry.